components/Atoms/Property.vue

```diff
@@ -1,20 +1,27 @@
 <template>
   <div class="property">
-    <label class="font-bold text-sm">{{ label }}</label>
-    <span v-if="href" class="text-sm text-underline"><NuxtLink :to="href">{{ value }}</NuxtLink></span>
+    <label class="font-bold text-sm">{{ $t(label) }}</label>
+    <span v-if="isLink" class="text-sm text-underline"><NuxtLink :to="value">{{ value }}</NuxtLink></span>
     <span v-else class="text-sm">{{ value }}</span>
   </div>
 </template>
 
 <script setup lang="ts">
-withDefaults(defineProps<{
+const props = withDefaults(defineProps<{
   label: string,
-  value?: string,
-  href?: string
+  value?: string
 }>(), {
-  href: '',
   value: ''
 });
+
+const isLink = computed(() => {
+  try {
+    return Boolean(new URL(props.value));
+  }
+  catch(e){
+    return false;
+  }
```

Comment:
Waarom moet dit in een trycatch? Is dit URL validatie? Dan misschien handiger om een prop validator toe te voegen

Reply:
Inmiddels besproken en daaruit blijkt dat de trycatch toch nodig is omdat de template moet weten of het een link is of niet